### DISCUSSION TITLE
Support for locale ?

### DISCUSSION THREAD

**Discussion by sskagemo** [pyscript/pyscript#1398] [General]
Currently, trying to set the locale so that I get numbers formatted as we do it Norway (10.000.000,00 instead of 10,000,000.00), gives an error:
```python
import locale
tall=10_000_000.00
locale.setlocale(locale.LC_MONETARY, 'nb_NO.utf8')
print(locale.currency(tall, symbol=False, grouping=True))
```

Result:
```
Traceback (most recent call last):
  File "/lib/python3.10/_pyodide/_base.py", line 460, in eval_code
    .run(globals, locals)
  File "/lib/python3.10/_pyodide/_base.py", line 306, in run
    coroutine = eval(self.code, globals, locals)
  File "<exec>", line 5, in <module>
  File "/lib/python3.10/locale.py", line 273, in currency
    raise ValueError("Currency formatting is not possible using "
ValueError: Currency formatting is not possible using the 'C' locale.
```

(I am new to this, so maybe this belongs to Pyodide, more than PyScript ... ?)

**Comment by hoodmane** [comment]
Thanks for the report @sskagemo!

> maybe this belongs to Pyodide, more than PyScript?

Yes, can you check whether the same reproduction works here:
https://pyodide.org/en/latest/console.html
If so, please open an issue on https://github.com/pyodide/pyodide . (copy pasting this report there would be perfect.)

**Reply by hoodmane** [reply]
I note that setting the locale isn't necessary to get the error. `locale.currency` always fails with this error,

**Reply by sskagemo** [reply]
sorry for the delay, but I have now opened an issue at pyodide. And thanks for the note that setting locale is not necessary for getting the error
